src/ipc/server.rs

```diff
@@ -146,6 +147,31 @@ impl Ipc {
                 }
             }
             Command::Ping => Response::Ok,
+            Command::SetVisible { bar_name, visible } => {
+                let windows = application.windows();
+                let found = windows
+                    .iter()
+                    .find(|window| window.widget_name() == bar_name);
+                if let Some(window) = found {
```

Comment:
Please add whitespace between lines:

```suggestion

                if let Some(window) = found {
```